modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -280,3 +280,53 @@ sudo ./gravity plan execute --phase=< insert phase > --force --debug
 ----
 
 If an error is returned, wait a few minutes and repeat the previous steps.
+
+== Roll Back a Cluster-Level Component (Appliance) Upgrade
+
+If you cannot resolve issues with an upgrade, you can roll back the upgrade steps that executed.
+
+. Use the `gravity plan` command to identify the phase in which the upgrade paused: 
++
+----
+sudo ./gravity plan
+----
++
+The following partial list provides upgrade phase examples:
++
+----
+* init 
+* checks
+* bootstrap
+  * node-1
+* masters
+  * node-1   
+    * drain
+    * system-upgrade
+    * taint
+    ...
+* runtime
+  * rbac-app
+  * site
+  * kubernetes
+* app
+  * telekube     
+----
+. Roll back the steps that completed in reverse order, as shown in the following example:
++
+----
+$ sudo gravity plan rollback --phase=/masters/node-1/taint
+$ sudo gravity plan rollback --phase=/masters/node-1/system-upgrade
+$ sudo gravity plan rollback --phase=/masters/node-1/drain
+...
+----
++
+A group of steps can also be rolled back:
++
+----
+$ sudo gravity plan rollback --phase=/masters
+----
+. After all steps are rolled back, mark the upgrade operation as failed:
```

Comment:
Nit: "After all steps are rolled back, update the plan as completed in order to mark it completely rolled-back:"